tasks/eth/fp-recovery/001-blacklist-game/README.md

```diff
@@ -0,0 +1,64 @@
+# Deputy Guardian - Blacklist Dispute Game
+
+Status: CONTINGENCY TASK, SIGN AS NEEDED
+
+## Objective
+
+This task executes a `blacklistDisputeGame` operation in the `OptimismPortalProxy` on behalf of the `Guardian` role. Blacklisting the dispute game prevents any withdrawals proven to be contained within the game's proposed output root from being finalized.
+
+The batch will be executed on chain ID `1`, and contains `1` transactions.
+
+## Tx #1: Blacklist Dispute Game
+
+Executes the `blacklistDisputeGame` call to the `OptimismPortalProxy`.
+
+**Function Signature:** `blacklistDisputeGame(address,address)`
+
+**To:** `0x4220C5deD9dC2C8a8366e684B098094790C72d3c`
```

Comment:
Just to make sure, This address is not yet deployed on L1 is that desired?

Reply:
Sorry this was copy-pasted from the Sepolia recovery tasks and I missed changing this address. This should be `0xc6901F65369FC59fC1B4D6D6bE7A2318Ff38dB5B` (the Mainnet `DeputyGuardianModule`) which would match the `to` value for the `input-template.json` for this task.

Reply:
That make more sense to me ty!

Reply:
Fixed